conformance/packages/dns-test/src/name_server.rs

```diff
@@ -223,6 +223,12 @@ impl NameServer<Stopped> {
         self
     }
 
+    /// Copy a file to the name server's filesystem
+    pub fn cp(&self, path: &str, contents: &str) -> Result<()> {
```

Comment:
This doesn't seem to get used in this commit, maybe move it into the next commit?